data/json/items/ammo.json

```diff
@@ -1041,10 +1041,10 @@
     "material": [ "coal" ],
     "phase": "solid",
     "volume": "200 ml",
-    "weight": "1500 mg",
-    "//": "Density somewhere around 1.5 g/cm3 (solid anthracite), roughly 50kJ per unit of coal",
+    "weight": "931 mg",
+    "//": "Density somewhere around 0.931 g/cm3 (bulk density of anthracite), roughly 31kJ per unit of coal",
```

Comment:
Why not just change the volume?
Unchanged mass = unchanged energy content = no need to adjust all of the requirements all over again.

Also 1.5g happened to exactly divide 37.5g, which is the value that I've been using -- based on [bugfix](https://github.com/CleverRaven/Cataclysm-DDA/pull/62180#issuecomment-1416346274) of the regression in #56162 -- so it would have been easier to resolve merge conflicts: just multiply ×25.

Reply:
Well, I didn't really like having charcoal at 8 ml, it was an awkward value compared to 5ml. And I didn't want coal to be 1.4 ml either, I'd rather use a 1ml fuel per coal.